src/app/(application)/meus-cupons/page.tsx

```diff
@@ -16,26 +21,34 @@ import Arrow from "@/images/easytolive/icons/arrow-next-right-primary.svg";
 interface IfilterOptions {
   id: string;
   name: string;
+  emptyText: string;
   icon: StaticImageData;
   textColor: string;
+  href?: string;
+  HrefLabel?: string;
 }
 
 const filterOptions = [
   {
     id: "ACTIVE",
     name: "cupons ativos",
+    emptyText: "Nenhum cupom ativo disponível",
+    href: "/",
+    HrefLabel: "Buscar descontos",
```

Comment:
```suggestion
    hrefLabel: "Buscar descontos",
```

Same here, that's consequence